assets/scss/base.scss

```diff
@@ -62,7 +62,7 @@ body {
 h1, h2, h3, h4, h5, h6 {
   color: $gray9;
   letter-spacing: -0.02rem;
-  font-weight: 700;
+  font-weight: bold;
```

Comment:
We only have two font weights: 400 and 700. So, to reduce possible confusion, it's better to call them "normal" and "bold" everywhere (except the font-face declarations above).